packages/patternfly-4/react-integration/README.md

```diff
@@ -1,7 +1,33 @@
-# react-integration
+# Patternfly 4 React Integration Testing
+
+## Overview
+Here you will find a demo application written in typescript, along with cypress integration testing.  This is used to allow developers to verify their code against a working react app that is running their latest PF4 react changes.
+
+New componets should be written in typescript and added to the demo app.  Contributors should write cypress tests to test the functionality of their components.  Contributors should also test all of the user facing interfaces and components that they are creating.
+
+## Building your react component
+After completing the development of their component a contributor should perform a build so they can start writing integration tests.
 
 1. Make your changes in packages/patternfly-4/react-core
-2. `yarn build` in packages/patternfly-4/react-core
-3. `yarn integration:setup` in packages/patternfly-4/react-integration
-4. Update the demo in packages/patternfly-4/react-integration/demo-app-ts/src/App.tsx
-5. `yarn build:demo` in packages/patternfly-4/react-integration/demo-app-ts
+2. Type `yarn build` to build your component.
+
+## Running the demo app
+To launch the demo app a contributor can run the following command:
+1. `yarn start demo:app` this will launch the demo app and watch for changes in it.
+
+## Building a typescript demo
+Prior to writing a cypress test you will need to build a demo page to test aginst.  Demos can be found in the [components demos section](https://github.com/patternfly/patternfly-react/tree/master/packages/patternfly-4/react-integration/demo-app-ts/src/components/demos).  You can create a new folder to add your demo.
+
+In order for your demo to show up in the demo app you should add it to the [Demo.ts](https://github.com/patternfly/patternfly-react/tree/master/packages/patternfly-4/react-integration/emo-app-ts/src/Demos.ts) file. Once you add it to the ts file you should see your demo show up in the navigation on the right hand side of the app.
+
+## Building and running your integration test
```

Comment:
Building Cypress tests
We use Cypress for integration testing. View all of our existing Cypress tests here.

1. Make sure your demo app is running. If you need to launch the demo application, run the following command:

`yarn start:demo-app`

2. Run the following command to launch Cypress:

`yarn start:cypress`

You can leave both the demo app and Cypress running while you develop your tests. You may receive notifications from Cypress that ask you to refresh as you make changes. Use the “Try again” button to refresh Cypress with your updated tests. 

To learn more about writing integration tests with Cypress, [view their docs](https://docs.cypress.io/guides/overview/why-cypress.html#In-a-nutshell).

Add any new integration tests you write to the [integration folder here](https://github.com/patternfly/patternfly-react/tree/master/packages/patternfly-4/react-integration/cypress/integration).

Reply:
Done, but I removed "To learn more about writing integration tests with Cypress, view their docs." since we have that information at the top.